test/parse_tests/test_parse_es.py

```
@@ -0,0 +1,189 @@
+    extract_duration, extract_datetime
+)
+
+
+class TestDatetime_es(unittest.TestCase):
+
+    def test_datetime_by_date_es(self):
+        # test currentDate==None
+        _now = datetime.now()
+        relative_year = _now.year if (_now.month == 1 and _now.day < 11) else \
+            (_now.year + 1)
+        self.assertEqual(extract_datetime_es("11 ene", anchorDate=_now)[0],
```

Ruff

Undefined name `extract_datetime_es`

(F821)